modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/matchers/CompletableFutureMatcher.java

```diff
@@ -53,15 +53,18 @@
      * Class of throwable that should be the cause of fail if the future should fail. If {@code null}, the future should be completed
      * successfully.
      */
-    private final Class<? extends Throwable> causeOfFail;
+    private final @Nullable Class<? extends Throwable> causeOfFail;
+
+    /** Fragment that must be a substring of a error message (if {@code null}, message won't be checked). */
+    private final @Nullable String errorMessageFragment;
```

Comment:
Since you added a new parameter, please add it into `describeMismatchSafely` or other appropriate place

Reply:
Try to fix it.